connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/util/RemoteSparkSession.scala

```diff
@@ -168,6 +159,56 @@ trait RemoteSparkSession extends ConnectFunSuite with BeforeAndAfterAll {
   import SparkConnectServerUtils._
   var spark: SparkSession = _
   protected lazy val serverPort: Int = port
+  protected lazy val transferredClientJarIfNeed: Boolean = {
+    if (sys.env.contains("USE_MAVEN")) {
+      if (RemoteSparkSession.transferredClientJar.get()) {
+        true
+      } else {
+        val fileOption = IntegrationTestUtils.tryFindJar(
+          "connector/connect/client/jvm",
+          // SBT passes the client & test jars to the server process automatically.
+          // So we skip building or finding this jar for SBT.
+          "sbt-tests-do-not-need-this-jar",
+          "spark-connect-client-jvm")
```

Comment:
`spark-connect-client-jvm` is not guaranteed to exist uuring Maven testing, so the test may also be skipped